config/install/rules.settings.yml

```diff
@@ -1,4 +1,2 @@
-log_errors: warning
-debug_log: false
-debug: false
-log_level: info
+log: false
+log_level: warning
```

Comment:
I think "debug_log" should stay as "debug_log". Let's rename "debug" to "debug_screen" or "debug_message" or "debug_html" to make it clear that it prints debug info to the current page. I like "debug_screen" most.